docs/getting_started.rst

```diff
@@ -237,8 +237,8 @@ Moderator
 A moderator is a step above the average users. You can set multiple moderator
 roles with the ``[p]set addmodrole`` and ``[p]set removemodrole`` commands.
 
-For example, in the mod cog (again), a mod will be able to mute, kick and ban;
-but he won't be able to modify the cog settings with the ``[p]modset`` command.
+For example, in the mutes cog, a mod will be able to mute;
+but he won't be able to modify the cog settings with the ``[p]muteset`` command.
```

Comment:
I don't know why that is, but this is currently not correct and some of the `[p]muteset` commands are allowed to be used by mods